tests/components/webostv/test_config_flow.py

```diff
@@ -128,13 +108,33 @@ async def test_options_flow_cannot_retrieve(hass: HomeAssistant, client) -> None
     """Test options config flow cannot retrieve sources."""
     entry = await setup_webostv(hass)
 
-    client.connect = Mock(side_effect=ConnectionRefusedError())
+    client.connect = AsyncMock(side_effect=ConnectionRefusedError())
     result = await hass.config_entries.options.async_init(entry.entry_id)
     await hass.async_block_till_done()
 
     assert result["type"] is FlowResultType.FORM
     assert result["errors"] == {"base": "cannot_retrieve"}
 
+    # recover
+    client.connect = AsyncMock(return_value=True)
+    result2 = await hass.config_entries.options.async_configure(
+        result["flow_id"],
+        user_input=None,
+    )
+    await hass.async_block_till_done()
+
+    assert result["type"] is FlowResultType.FORM
+    assert result["step_id"] == "init"
```

Comment:
I think I now found a good argument on why to use `result` instead of `resultn` :)